gen/build_deploy/bash.py

```diff
@@ -638,12 +640,18 @@ def fill_template(base_name, format_args):
 def make_dcos_launch():
     # NOTE: this needs to be kept in sync with build_dcos_launch.sh
     work_dir = py.path.local.mkdtemp()
+    # the tree will not be in the working dir if run from downstream
```

Comment:
upstream is dcos/docs, downstream has a treeinfo.json that points to an upstream.